client/hawc_client/epiv2.py

```diff
@@ -0,0 +1,538 @@
+import pandas as pd
+from requests import Response
+
+from .client import BaseClient
+
+
+class EpiV2Client(BaseClient):
+    """
+    Client class for epidemiology v2 requests.
+    """
+
+    """
```

Comment:
uncomment these methods for metadata and data?

Reply:
done